src/maintenance/azext_maintenance/azext_metadata.json

```diff
@@ -1,3 +1,4 @@
 {
+    "azext.isExperimental": true,
```

Comment:
In previous release it’s stable. Do you want to convert it to experimental?

Reply:
It private preview

Reply:
we are releasing new feature , earlier was GA VMSS support.